lint-staged.config.mjs

```diff
@@ -1,5 +1,5 @@
 export default {
   '*.(json|md|ts|mjs)': 'prettier --write',
   '*.(ts|mjs)': 'eslint --max-warnings 0 .',
-  '*.ts': () => 'tsc',
+  '*.ts': () => 'npm run prepare:esbuild',
```

Comment:
This change does not seem related. It should be in a separate PR.

Also, I note that none of our other projects have a lint-staged rule to run esbuild. What is the rationale?

Reply:
Because this file was added, a bunch of .js files get created and then when you try to run this if breaks because it cant find the right functions from other files. 

So it is either this, or we remove `'*.ts': () => 'tsc',`  from the config file so nothing builds on commit. I actually would prefer that

Reply:
See https://github.com/nasa-gcn/architect-plugin-dynamodb-local/pull/43.

Reply:
Ah this makes so much sense. Okay perfect, I will have these removed and updated in the next commit